src/mammos_analysis/kuzmin.py

```diff
@@ -251,18 +260,34 @@ def plot(
         self,
         T: mammos_entity.Entity | astropy.units.Quantity | numpy.ndarray | None = None,
         ax: matplotlib.axes.Axes | None = None,
+        Celsius: bool = False,
         **kwargs,
     ) -> matplotlib.axes.Axes:
-        """Plot A as a function of temperature using Kuzmin formula."""
+        """Plot A as a function of temperature using Kuzmin formula.
+
+        Args:
+            T: If specified, the exchange stiffnedd is plotted against this array.
+                Otherwise, a uniform array of 100 points is generated between the
+                minimum and the maximum available data.
+            ax: optional matplotlib ``Axes`` instance to plot on an existing subplot.
+            Celsius: If True, plots the temperature in degree Celsius.
+            **kwargs: Additional plotting arguments.
+        """
         if not ax:
             _, ax = plt.subplots()
         if T is None:
             T = np.linspace(min(self._T.value), max(self._T.value), 100)
         if not isinstance(T, me.Entity):
             T = me.T(T)
         A = self(T)
-        ax.plot(T.q, A.q, **kwargs)
-        ax.set_xlabel(T.axis_label)
+        if Celsius:
```

Comment:
Being paranoid: We are never checking if `Celsius` is actually a boolean right? If I pass `Celsius="(°C)"` (which is dumb 🤡), it will still create the plots in degree Celsius right?